src/nonlinear_to_conic.jl

```diff
@@ -255,7 +250,4 @@ MathProgBase.getsolution(m::NonlinearToConicBridge) = m.solution[1:size(m.A_ini,
 MathProgBase.getsolvetime(m::NonlinearToConicBridge) = MathProgBase.getsolvetime(m.nlp_model)
 
 MathProgBase.numvar(m::NonlinearToConicBridge) = m.numVar
-function MathProgBase.numconstr(m::NonlinearToConicBridge)
-    # is numconstr well defined for conic models?
```

Comment:
OK I thought it was well enough defined because there is a way to query number of constraints in conic models